docs/magent/adversarial_pursuit.md

```diff
@@ -50,6 +50,21 @@ binary_agent_id(extra_features=True)| 10
 one_hot_action(extra_features=True)| 9/Prey,13/Predator
 last_reward(extra_features=True)| 1
 
+### State space
+
+The observation space is a 45x45 map. It contains the following channels, which are (in order):
+
+name | number of channels
+--- | ---
+obstacle map| 1
+prey_presence| 1
+prey_hp| 1
+predator_presence| 1
+predator_hp| 1
+binary_agent_id(extra_features=True)| 10
+one_hot_action(extra_features=True)|  13 (max action space)
+last_reward(extra_features=True)| 1
+agent_position(minimap_mode=True)| 2
```

Comment:
So having the agent_position in the global map does not make a lot of sense to me. The position is already encoded by the position on the map, so it is redundant.

I'm also unsure why a global critic would want to concern itself with any of the extra features. But I don't have a serious complaint either.

Reply:
"I'm also unsure why a global critic would want to concern itself with any of the extra features. "

Can you explain? That seems like something documentation needs to cover

Reply:
I probably should have left this as a general comment, rather than a comment in the code. 

I'm just wondering if we really need to have binary agent IDs, or one_hot actions, etc in the state at all, even if the extra_features flag is set to true.

Reply:
If you are training only with the state observations wouldn't it be helpful?

Reply:
I don't think so? @justinkterry Any thoughts?

Reply:
Only training with state observations is not a use case that should be supported IMO